javascript/ql/lib/semmle/javascript/security/dataflow/ReflectedXssCustomizations.qll

```diff
@@ -25,28 +25,43 @@ module ReflectedXss {
    * is to prevent us from flagging plain-text or JSON responses as vulnerable.
    */
   class HttpResponseSink extends Sink instanceof Http::ResponseSendArgument {
-    HttpResponseSink() { not exists(getANonHtmlHeaderDefinition(this)) }
+    HttpResponseSink() { not exists(getAXssSafeHeaderDefinition(this)) }
   }
 
   /**
-   * Gets a HeaderDefinition that defines a non-html content-type for `send`.
+   * Gets a HeaderDefinition that defines a XSS safe content-type for `send`.
    */
-  Http::HeaderDefinition getANonHtmlHeaderDefinition(Http::ResponseSendArgument send) {
+  Http::HeaderDefinition getAXssSafeHeaderDefinition(Http::ResponseSendArgument send) {
```

Comment:
Could you keep the old versions around as `deprecated` predicates?  
`Customizations.qll` files should follow our deprecation policy.

Reply:
Sure, let me know if the changes are correct

Reply:
They look good 👍